company/principles.md

```diff
@@ -19,3 +19,16 @@ Node-RED has an open governance model which means everyone has a seat at the
 table. We will continue to champion that model and work with the community to
 ensure anyone who wants to contribute to the project is able to.
 
+## Everybody Codes
```

Comment:
I'm not sold on this. What does this mean? Everybody needs to know Git? To be honest, if notion had a decent approval flow, why stick with Git for the handbook?

Looking at how much time it cost to learn Git and be effective with it, I'd rather move our tooling to more user friendly ones. The implementation of this also feels at odds with the low-code approach Node-RED champions currently. If our core audience cannot contribute, why do we want this?

Reply:
Currently, this is our policy. Anyone wanting to contribute to the Handbook, needs to know how to use Git. Until our Handbook changes, then this is the case.

The articles from PostHog make good cases for how well it has worked there. GitHub is low-code for Handbook edits anyway (at least at single file level), as no need for cmd line, or local IDE.

Reply:
It's also about ensuring colleagues that they should be supported to code too. Anyone at FlowForge should feel empowered to edit Website or Handbook content if they so wish.

Rob's guide to Git in the handbook (which this references) is a great starting point to that attitude.

Reply:
I'd suggest simplifying this to everyone learns how to use the basics of _GitHub_ i.e. the software; at the most basic that's how to edit a file and go through the approvals process (a pull request), the UI itself does a pretty good job of hiding away most of the complexity.